documentation/book/proc-deploying-example-clients.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// assembly-hello-world.adoc
+
+[id='deploying-example-clients-{context}']
+=== Deploying example clients
+
+.Prerequisites
+* Before deploying a example client, the Kafka Cluster must be deployed.
+
+.Procedure
+
+Deploying a client is done by a single command. This command contains a few arguments which you may need to adjust.
```

Comment:
change to "You can deploy the client in a single command. This is an example command you must substitute `_test_` for your topic and `_my-cluster-kafka-bootstrap:9092_`  by your server address for both producer and consumer."